airflow/contrib/operators/gcp_vision_operator.py

```diff
@@ -0,0 +1,599 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from google.api_core.exceptions import AlreadyExists
+
+from airflow.contrib.hooks.gcp_vision_hook import CloudVisionHook
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class CloudVisionProductSetCreateOperator(BaseOperator):
+    """
+    Creates a new ProductSet resource.
+
+    :param product_set: (Required) The ProductSet to create. If a dict is provided, it must be of the same
+        form as the protobuf message `ProductSet`.
+    :type product_set: dict or google.cloud.vision_v1.types.ProductSet
+        https://googleapis.github.io/google-cloud-python/latest/vision/gapic/v1/types.html#google.cloud.vision_v1.types.ProductSet
+    :param location: (Required) The region where the ProductSet should be created. Valid regions
+        (as of 2019-02-05) are: us-east1, us-west1, europe-west1, asia-east1
+    :type location: str
+    :param project_id: (Optional) The project in which the ProductSet should be created. If set to None or
+        missing, the default project_id from the GCP connection is used.
+    :type project_id: str
+    :param product_set_id: (Optional) A user-supplied resource id for this ProductSet.
+        If set, the server will attempt to use this value as the resource id. If it is
+        already in use, an error is returned with code ALREADY_EXISTS. Must be at most
+        128 characters long. It cannot contain the character /.
+    :type product_set_id: str
+    :param retry: (Optional) A retry object used to retry requests. If `None` is
+        specified, requests will not be retried.
+    :type retry: google.api_core.retry.Retry
+        https://googleapis.github.io/google-cloud-python/latest/core/retry.html#google.api_core.retry.Retry
```

Comment:
Nie należy dodawać linków, ale włączyć wewnętrzne linkowanie.Tutaj przykład: https://github.com/apache/airflow/pull/4655

Reply:
Sądzę, że nie jest to póki co konwencja w Airflow, więc na razie zostawię tak, jak jest.
Jak wejdą Twoje zmiany to można to wyedytować.